### DISCUSSION TITLE
Can't add my helm chart repository because it is already used

### DISCUSSION THREAD

**Discussion by ronahk** [artifacthub/hub#2621] [Q&A] [answered]
I’m trying to add a public helm-repository on github to my artifacthub organization. When i enter the repository url i get an error message saying “There is another repository using this url”. I searched and saw that some user (i don't know that user nor it is related to my organization) uploaded my chart to artifacthub and indeed the repository is already in use. What can I do?

**Comment by tegioz** [comment, marked as answer]
Hi @ronahk 👋 

You can [claim its ownership](https://artifacthub.io/docs/topics/repositories/#ownership-claim) 🙂

**Reply by ronahk** [reply]
Thanks @tegioz for replying :)
So I add the artifacthub-repo.yml without the repositoryId (since I can't create one) and with the owner and it should work?
How do I later move it to my artifacthub organization?

**Reply by tegioz** [reply]
No worries! You can do it from the control panel. The link I shared above contains some more details about the process :)

**Reply by ronahk** [reply]
Great thanks a lot!
